alvr/dashboard/src/dashboard/mod.rs

```diff
@@ -80,7 +80,6 @@ impl Dashboard {
 
         // Audio devices need to be queried early to mitigate buggy/slow hardware queries on Linux.
         data_sources.request(ServerRequest::GetSession);
-        #[cfg(not(target_os = "linux"))]
         data_sources.request(ServerRequest::GetAudioDevices);
```

Comment:
later this will get removed completely for linux? (since we would remove cpal from it)

Reply:
well I prefer keeping it since it's needed for other platforms. the current flow should work for linux too even if more complicated than needed.

Reply:
it just seems kind of unnecessary and it makes calls to alsa for no reason on linux

Reply:
Ok then, we can readd the cfg in this position

Reply:
Actually, I prefer keeping it this way. There was a bug before where the presets were created but the values were not initialized immediately. You saw that the values were actually updated but it probably was because of a later session event, but not explicitly enforced. keeping as much codeunchanged is better IMO.

Reply:
Why not just keep the cfg

Reply:
I just explained

Reply:
wasn't it caused by just default configurations being wrong?

Reply:
Yea i know i was writing the comment when you sent that

Reply:
Yes, it's supposed to load the updated values at every startup. the call to update the value was missing

Reply:
I we could fix that separately but it's piling more and more platform specific code and that was starting to become spaghetti code.

Reply:
I double checked if any audio devices being created (there are none) and yes, i forgot that this part also makes loading beforehand (not just for cpal), so i would keep it, yes